src/petals/server/server.py

```diff
@@ -698,7 +698,7 @@ def run(self) -> None:
 
             delay = self.update_period - (time.perf_counter() - start_time)
             if delay < 0:
-                logger.warning("Declaring blocs to DHT takes more than --update_period, consider increasing it")
+                logger.warning(f"Declaring blocks to DHT takes more than --update_period, consider increasing it (currently {self.update_period})")
```

Comment:
As required by black (the style checker):

```suggestion
                logger.warning(
                    f"Declaring blocks to DHT takes more than --update_period, consider increasing it (currently {self.update_period})"
                )
```